### PR TITLE
Documentation: Unbreak build instructions after d0427b610c52a88f2e

### PR DESCRIPTION
**Pull request by nico**

### CODE REVIEW

File: Documentation/BuildInstructions.md [hunk 1]
```diff
@@ -172,7 +172,7 @@ Notes:
 - Installing osxfuse for the first time requires enabling its system extension in System Preferences and then restarting your machine. The output from installing osxfuse with brew says this, but it's easy to miss.
 - bash is needed because the default version installed on macOS doesn't support globstar
 - If you install some commercial EXT2 macOS fs handler instead of osxfuse and fuse-ext2, you will need to `brew install e2fsprogs` to obtain `mke2fs` anyway.
-- As of 2020-08-06, you might need to tell the build system about your newer host compiler. Once you've built the toolchain, navigate to `Build/i686/`, `rm -rf *`, then run `cmake ../.. -G Ninja -DCMAKE_C_COMPILER=gcc-10 -DCMAKE_CXX_COMPILER=g++-10`, then continue with `ninja install` as usual.
+- As of 2020-08-06, you might need to tell the build system about your newer host compiler. Once you've built the toolchain, navigate to `Build/i686/`, `rm -rf *`, then run `cmake ../../.. -G Ninja -DCMAKE_C_COMPILER=gcc-10 -DCMAKE_CXX_COMPILER=g++-10`, then continue with `ninja install` as usual.
```
Comment (alimpfard) [inline Documentation/BuildInstructions.md:175]: :thinking: wouldn't that go out of $SERENITY_ROOT?
`foo/$SERENITY_ROOT/Build/i686/../../..` would be `foo` unless I'm being really dumb

Comment (gunnarbeutner) [reply Documentation/BuildInstructions.md:175]: Btw, #8594 is going to remove this entirely. :)

**Comment by nico** [comment]
closed in favor of #8594

File: Documentation/BuildInstructions.md (hunk 1 above)
Comment (nico) [reply Documentation/BuildInstructions.md:175]: I followed the build instructions (...before this got merged) and ended up in `Toolchain/Build/i686`. Maybe that's different in the new instructions, but the instructions as of when I made this patch created the Build dir below the Toolchain dir for whatever reason.

Comment (alimpfard) [reply Documentation/BuildInstructions.md:175]: Huh.
That bit is pretty odd

**Pull request closed without merging**